project/build.scala

```diff
@@ -206,7 +207,10 @@ object build extends Build {
         case ex: Exception if ex.getMessage.contains(nothingToCommit) => println(nothingToCommit)
       }
     },
+    // TODO: doesn't work at the moment, see https://github.com/sbt/sbt-pgp/issues/42
+    publishSigned := publish.value,
```

Comment:
we aggregate just the bits we want to publish https://github.com/ensime/ensime-server/blob/2.0/project/EnsimeBuild.scala#L243

note also that if you have a custom test configuration, with its own dependencies, you must add them to `test` and your config or it triggers a bug in ivy that means your `main` config will depend on those jars. Probably not relevant for you but a major source of pain for us.